Source/WebKit/UIProcess/gtk/ViewGestureControllerGtk.cpp

```diff
@@ -195,10 +195,11 @@ bool ViewGestureController::SwipeProgressTracker::handleEvent(PlatformGtkScrollD
     }
 
     Seconds time = Seconds::fromMilliseconds(eventTime);
-    if (time != m_prevTime)
+    if (time > m_prevTime) {
```

Comment:
This change is intended to be part of this MR, right?

Reply:
Correct. When the event is null, we get timestamp 0 and don't want it here.

Reply:
Marking as resolved, feel free to unresolve if you think it warrants a comment.